docs/specviz/notebook.rst

```diff
@@ -1,5 +1,19 @@
 ***********************************
-Using SpecViz in a Jupyter Notebook 
+Using Specviz in a Jupyter Notebook 
 ***********************************
 
-Words go here...
+Specviz is developed to fully support analyzing spectra within your existing `Jupyter notebooks <https://jupyter.org/>`_! To use Specviz, install jdaviz in your notebook's python environment and add a new cell wherever you would like to use Specviz.
+
+    >>> # Import specviz
+    >>> from jdaviz import SpecViz
+    >>> # Instantiate an instance of Specviz
+    >>> specviz = SpecViz()
+    >>> # Display Specviz
+    >>> specviz.app
+
+To extract the data currently loaded into the viewer:
+::
+
+    specviz.app.get_data_from_viewer('spectrum-viewer')
```

Comment:
I think it's better to highlight the helpers since that's what it's for:
```suggestion
    specviz.get_spectra()
```
or maybe highlight both options?:
```suggestion
    specviz.get_spectra()

or use the lower-level glue interface::
     specviz.app.get_data_from_viewer('spectrum-viewer')
```